addons/im_livechat/models/discuss_channel_member.py

```diff
@@ -41,3 +41,18 @@ def _get_partner_data(self, fields=None):
                 } if self.partner_id.country_id else False
             return data
         return super()._get_partner_data(fields=fields)
+
+    def _allow_sending_seen_infos_to_channel(self):
+        """ Override to allow sending visitors seen infos to live chat channels"""
+        if self.channel_id.channel_type != "livechat":
+            return super()._allow_sending_seen_infos_to_channel()
+        operators = [self.channel_id.livechat_operator_id, self.channel_id._get_human_operator()]
```

Comment:
How does it work when the operator invites another one?

Reply:
It does not work. Maybe we should keep the compromise described in: https://github.com/odoo/odoo/pull/142546#discussion_r1409023675